### PR TITLE
make cfn json template consistent with its yaml equivalent

### PR DESCRIPTION
**Pull request by borjatur**
*Issue #, if available:*

*Description of changes:*

JSON cfn template is no longer working because of python2.7 runtime has been deprecated, this PR is for updating the template as well as the source code for the one available in the yaml cf template which is compatible with python3.7 runtime

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: slack-notifier/cfn-templates/slack-notifier.json [hunk 1]
```diff
@@ -71,49 +73,10 @@
...
...rllib.request import Request, urlopen, URLError, HTTPError\nfrom urllib.parse import urlencode\n# Setting up logging\nlogger = logging.getLogger()\nlogger.setLevel(logging.INFO)\n# main function\ndef handler(event, context):\n    message =  str(\n        event['detail']['eventDescription'][0]['latestDescription']  +\n        \"\\n\\n<https://phd.aws.amazon.com/phd/home?region=us-east-1#/event-log?eventID=\" +\n        event['detail']['eventArn'] +\n        \"|Click here> for details.\"\n    )\n    json.dumps(message)\n    slack_message = {\n      \"channel\": \"${SlackChannel}\",\n      \"text\": message,\n      \"username\": \"AWS - Personal Health Updates\"\n    }\n    logger.info(str(slack_message))\n    req = Request(\n      \"${HookURL}\",\n      data=json.dumps(slack_message).encode(\"utf-8\"),\n      headers={\"content-type\": \"application/json\"}\n    )\n    try:\n      response = urlopen(req)\n      response.read()\n      logger.info(\"Message posted to: %s\", slack_message['channel'])\n    except HTTPError as e:\n       logger.error(\"Request failed : %d %s\", e.code, e.reason)\n    except URLError as e:\n       logger.error(\"Server connection failed: %s\", e.reason)\n"
```
Comment (tipuq) [inline slack-notifier/cfn-templates/slack-notifier.json:76]: Something went wrong with indentation and formatting here, could you fix before me we merge?

Comment (tipuq) [reply slack-notifier/cfn-templates/slack-notifier.json:76]: I tested and it looks ok, but have a look at the formatting of https://github.com/aws/aws-health-tools/blob/master/slack-notifier/cfn-templates/slack-notifier.yml#L52

Comment (borjatur) [reply slack-notifier/cfn-templates/slack-notifier.json:76]: sure, back to the keyboard in some minutes

Comment (borjatur) [reply slack-notifier/cfn-templates/slack-notifier.json:76]: I'm on it, yaml offers much better multiline string representation , the 1:1 equivalent between yaml and json is what I've suggested here but I will replace "Fn::Sub" for "Fn::Join" for improving readability allowing the previous multiline approach

**Pull request merged**